### PR TITLE
Add conf to host, move plugin to sbin

### PR DESCRIPTION
**Pull request by zbud-msft**
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -323,6 +323,10 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-db-cli_*.deb || \
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-rsyslog-plugin_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Generate host conf for rsyslog_plugin
```
Comment (renukamanavalan) [inline files/build_templates/sonic_debian_extension.j2:326]: You can create this file directly into $FILESYSTEM_ROOT_ETC/rsyslog.d/host_events.conf
Avoid the mv

**Review by renukamanavalan — APPROVED** [review]

File: files/build_templates/events_info.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "yang_module": "sonic-events-host",
+    "proclist": [
+        {
+            "name": "host",
```
Comment (renukamanavalan) [inline files/build_templates/events_info.json]: The program name in the message is not "host", but "monit"

File: files/build_templates/host_regex.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "params": [ "fs", "usage", "limit" ]
+    },
+    {
+        "tag": "memory-usage",
+        "regex": "(?:mem|cpu user) usage of (\d+\.\d+)% matches resource limit .(?:mem|cpu user) usage>(\d+\.\d+)%.",
+        "params": [ "usage", "limit" ]
+    },
+    {
+        "tag": "event-sshd",
+        "regex": "auth fail: ([a-zA-Z0-9-_\s]*).",
```
Comment (renukamanavalan) [inline files/build_templates/host_regex.json]: You need to look for "auth fail: Password incorrect. " 
This is a special case. This is not fired for incorrect password, but the internal system state blocks from creating this user.

File: files/build_templates/host_regex.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "tag": "disk-usage",
+        "regex": ".([a-zA-Z0-9-_]*). space usage (\d+\.\d+)% matches resource limit .space usage.(\d+\.\d+)%.",
+        "params": [ "fs", "usage", "limit" ]
+    },
+    {
+        "tag": "memory-usage",
+        "regex": "(?:mem|cpu user) usage of (\d+\.\d+)% matches resource limit .(?:mem|cpu user) usage>(\d+\.\d+)%.",
```
Comment (renukamanavalan) [inline files/build_templates/host_regex.json]: Report only mem usage. 
Need a different tag for cpu usage
If we don't have an event, you can create one

File: files/build_templates/host_regex.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "tag": "disk-usage",
```
Comment (renukamanavalan) [inline files/build_templates/host_regex.json]: proc.name is monit for disk & mem usage. sshd for auth fail.
You would need to split into 2 files (monit-regex, sshd-regex) and have two entries in  the events_info.json

File: files/build_templates/sonic_debian_extension.j2 (hunk 1 above)
Comment (zbud-msft) [reply files/build_templates/sonic_debian_extension.j2:326]: Tried to do this, but run into permission issues

Comment (renukamanavalan) [reply files/build_templates/sonic_debian_extension.j2:326]: You may have to run the j2 as sudo
It is ok, either way

Comment (zbud-msft) [reply files/build_templates/sonic_debian_extension.j2:326]: Tried as sudo but still same error, not sure

**Review by renukamanavalan — APPROVED** [review]

**Pull request merged**